kieserver/image.yaml

```diff
@@ -89,8 +89,8 @@ envs:
       example: "false"
       description: "Set if the EJB timer datasources should be auto configured, if false, user will need to define the default and the ejb datasources using the DATASOURCES or DB_SERVICE_PREFIX_MAPPING environment variables manually. Default is true"
     - name: "KIE_SERVER_STARTUP_STRATEGY"
-      example: "LocalContainersStartupStrategy"
-      description: "When set to LocalContainersStartupStrategy, allows KIE server to start up and function with local config, even when a controller is configured and unavailable"
+      example: "OpenShiftStartupStrategy | LocalContainersStartupStrategy"
```

Comment:
Maybe would be better to pot here just OpenShiftStartupStrategy or LocalContainersStartupStrategy.
This example looks like there may be several strategies available at the same time, may confuse users.

Reply:
Good point. Leave only OpenShiftStartupStrategy as an example.